test/controllers/TransitAccompanyingDocumentControllerSpec.scala

```diff
@@ -61,6 +61,7 @@ class TransitAccompanyingDocumentControllerSpec
     def applicationBuilder: GuiceApplicationBuilder =
       new GuiceApplicationBuilder()
         .configure(Configuration("metrics.enabled" -> "false"))
+        .overrides(bind(classOf[TransitAccompanyingDocumentController]).to(classOf[LiveTransitAccompanyingDocumentController]).eagerly())
```

Comment:
Is this something that should be done globally for `test` in `test.application.conf`?

Reply:
This is only used in this particular test, and will be removed once the TAD story is played